enhancements/baremetal/enable-baremetal-on-other-platforms.md

```diff
@@ -0,0 +1,197 @@
+---
+title: enable-baremetal-on-other-platforms
+authors:
+  - "@asalkeld"
+  - "@sadasu"
+reviewers:
+  - "@hardy"
+  - "@romfreiman"
+  - "@dhellman"
+approvers:
+  - "@hardy"
+creation-date: 2021-08-20
+last-updated: 2021-08-20
+status: implementable
+see-also:
+  - "/enhancements/baremetal-provisiong-config.md"
+replaces:
+superseded-by:
+---
+
+# Enable baremetal on other Platforms to support centralized host management
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Baremetal Host API is only available when deploying an OpenShift cluster with the baremetal
+platform (via the IPI or AI (Assisted Installer) workflow). Having the ability to
+manage baremetal hosts from clusters without requiring the cluster to be on baremetal
+would be beneficial to customers.
+
+## Motivation
+
+An initial driver of this feature are the centralized host management use cases
+in edge topologies, which without this feature, is restricted to having the
+central OpenShift cluster deployed on baremetal.
+
+See:
+- https://github.com/openshift/enhancements/blob/master/enhancements/installer/agent-based-installation-in-hive.md
+- https://github.com/openshift/assisted-service/tree/master/docs/hive-integration
+
+### Goals
+
+The specific goals of this proposal are to:
+
+Support the centralized host management use case by partially enabling Baremetal Host API
+on the following platforms:
+- None
+- OpenStack
+- vSphere
+
+We will be successful when:
+
+Centralized host management can deploy clusters when running on the above platforms.
+
+### Non-Goals
+
+Allow Baremetal Host API to be fully enabled on all platforms.
+
+Note that the scope of this epic is only enablement of the Metal3
+BaremetalHost API and associated controller/services - the centralized host management flow
+currently interacts directly via this API without any Machine API integration.
```

Comment:
We could be even more explicit here to avoid confusion e.g something like "... without any Machine API integration.  This means further work will be required via a future enhancement to enable the single cluster case where a combination of e.g VM controlplane and Baremetal workers is desired"

Reply:
cool, i'll add that